lib/notification.php

```diff
@@ -14,6 +14,15 @@ public static function sendMail($path){
 			return;
 		}
 		$email = \OCP\Config::getUserValue(\OCP\User::getUser(), 'settings', 'email', '');
+		if (\OCP\App::isEnabled(user_ldap)){
```

Comment:
we can do this without the check but I thought It would be better to check first then have a statement $user  = \OCP\Config::getUserValue(\OCP\User::getUser(), 'user_ldap', 'displayName', ''); which is always empty (aka fails) if user_ldap is not enabled. But I agree the behavior is the same